react/features/deep-linking/constants.js

```diff
@@ -0,0 +1,6 @@
+/**
+ * The namespace of the i18n/translation keys.
+ *
+ * @type {string}
+ */
+export const _TNS = 'deepLinking';
```

Comment:
Do you think this is useful to have as a constant? I ask because we haven't worried about this before. You might be better off just making this a function instead, too.

Reply:
Why function?

Reply:
Because current usage all does `${_TNS}.string`. If there's gonna be an abstraction to potentially remove duplication, might as well remove the string creation duplication, is my thinking. I'm more interested in why you found having the string namespace to be useful when that pattern hasn't been used widely.